render-config.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+VAULT_TOKEN=${1:-$(cat $HOME/.vault-token)}
+DSDE_TOOLBOX_DOCKER_IMAGE=broadinstitute/dsde-toolbox:consul-0.20.0
+VAULT_SERVICE_ACCOUNT_PATH=secret/dsde/terra/kernel/integration/toolsalpha/crl_janitor/app-sa
+VAULT_CLIENT_SERVICE_ACCOUNT_PATH=secret/dsde/terra/kernel/integration/toolsalpha/crl_janitor/client-sa
+SERVICE_ACCOUNT_OUTPUT_FILE_PATH="$(dirname $0)"/src/test/resources/rendered/sa-account.json
+CLIENT_SERVICE_ACCOUNT_OUTPUT_FILE_PATH="$(dirname $0)"/src/test/resources/rendered/client-sa-account.json
+
+docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN ${DSDE_TOOLBOX_DOCKER_IMAGE} \
+            vault read -format json ${VAULT_SERVICE_ACCOUNT_PATH} \
+            | jq -r .data.key | base64 -d > ${SERVICE_ACCOUNT_OUTPUT_FILE_PATH}
+docker run --rm --cap-add IPC_LOCK \
+            -e VAULT_TOKEN=$VAULT_TOKEN ${DSDE_TOOLBOX_DOCKER_IMAGE} \
+            vault read -format json ${VAULT_SERVICE_ACCOUNT_PATH} \
```

Comment:
should this be VAULT_CLIENT_SERVICE_ACCOUNT?

Reply:
Done. The test passed because carjaniror-sa has project level pubsub admin permissions, so we can use that to publish message as well.
https://github.com/broadinstitute/terraform-ap-modules/blob/master/crl-janitor/variables.tf#L45
I saw that last week and planed to check with you on reason for this permission. I remembered you need it for some stairway pubsub stuff, but I don't know the detail